packages/calcite-components/src/components/date-picker-day/date-picker-day.scss

```diff
@@ -25,9 +24,10 @@
     w-full
     relative;
   line-height: var(--calcite-font-line-height-fixed-base);
-  background: none;
   block-size: var(--calcite-internal-day-size);
   outline-color: var(--calcite-color-transparent);
+  background: var(--calcite-date-picker-day-background-color, none);
```

Comment:
Should this be `background-color` so that the token isn't allowed to do other background things? I wonder why `none` is even necessary?

Reply:
It should be `background-color` and default value isn't required.